homework/index.js

```diff
@@ -30,13 +30,103 @@
     return elem;
   }
 
+  function removeChildren(element) {
+    while (element.firstChild) {
+      element.removeChild(element.firstChild);
+    }
+  }
+
   function main(url) {
```

Comment:
1. The assignment asked for the first repository to be displayed when your application starts up. This is not happening in your case. 
2. Your `main` function is fairly large, which makes it difficult (for others and yourself) to maintain. Your code could benefit from a more granular approach, with a couple of functions for dedicated tasks.

Reply:
I've shorten my main function at week2's homework. I understood, thanks Jim.